packages/circle-demo-webapp/app/routes/wallet.$id/components/WalletSendDialog.tsx

```diff
@@ -25,7 +26,7 @@ import {
   DialogTrigger,
 } from '~/components/ui/dialog';
 import { CircleError, ErrorResponse } from '~/lib/responses';
-import { isAddress } from '~/lib/utils';
+import { getExplorerUrl, isAddress } from '~/lib/utils';
```

Comment:
I might have deleted `getExplorerUrl` helper from the demo app in my latest merged PR, as it was not used outside of Transaction components, but now we need it here, so bring it back please, sorry.

Reply:
I took care of it already in your branch